server/boot/user.js

```diff
@@ -631,4 +645,51 @@ module.exports = function(app) {
       return res.render('account/forgot');
     });
   }
+
+  function getReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    return res.render('account/report-profile', {
+      title: 'Report User',
+      username
+    });
+  }
+
+  function postReportUserProfile(req, res) {
+    const username = req.params.username.toLowerCase();
+    const report = req.body.reportDescription;
+    const { user } = req;
+
+    return Email.send$({
+      type: 'email',
+      to: 'Team@FreeCodeCamp.com',
+      cc: user.email,
+      from: 'Team@FreeCodeCamp.com',
+      subject: 'Abuse Report : Reporting ' + username + '\'s profile.',
+      text: dedent(`
+        Hello Team,\n
+        This is to report the profile of ${username}.\n
+        Report Details:\n
+        ${report}\n\n
+        Reported by:
+        Username: ${user.username}
+        Name: ${user.name}
+        Email: ${user.email}\n
+        Thanks and regards,
+        ${user.name}
+      `)
+    }, err => {
+      if (err) {
```

Comment:
errors should be sent to the default express handler using `return next(err)`. This prevents having to write duplicate code.

You can still define the redirect by adding the `redirectTo` property to the error instance.

https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/server/middlewares/error-handlers.js#L32